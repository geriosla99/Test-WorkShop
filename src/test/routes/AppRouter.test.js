import React from "react";
import {AppRouter} from "../../routers/AppRouter";
import { mount } from "enzyme";
import { AuthContext } from '../../auth/AuthContext'
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

describe('Pruebas en <AppRouter/>', () => {

    const contextValue ={
        dispatch: jest.fn(),
        user:{
            logged: true
        }
    }    

    
    test('Mostrar login si no esta autenticado', () => {

        Enzyme.configure({ adapter: new Adapter() })

       
        const wrapper = mount(
            <AuthContext.Provider value ={contextValue}>
                <AppRouter/>
            </AuthContext.Provider>
        )
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('.navbar').exists()).toBe(true)
    })
    

})
