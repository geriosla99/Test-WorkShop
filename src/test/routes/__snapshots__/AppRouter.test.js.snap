// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AppRouter/> Mostrar login si no esta autenticado 1`] = `ReactWrapper {}`;
